### DISCUSSION TITLE
How do I get sky cover (%) for weather observations?

### DISCUSSION THREAD

**Discussion by sthapa123** [weather-gov/api#221] [Q&A]
Using the weather gov api, I get the sky cover in percentage for weather forecast. However, I get the weather observations in values like FEW, BKN, SCT, etc. Is there any way to convert these values to sky cover percentage? I'd be very grateful if you could help me on this.

**Comment by hluker** [comment]
@sthapa123,
There isn't a direct/specific percentage for the weather observations. The METARs will provide cloud cover in either FEW (1/8 to 2/8 cloud coverage), SCT (scattered, 3/8 to 4/8 cloud coverage, BKN (broken, 5/8-7/8 coverage), and OVC (overcast, 8/8 coverage), along with their respective heights.

**Comment by britt-camus** [comment]
It turns out that `skyCover` is the field that has percentage-based coverage. See https://github.com/weather-gov/api/discussions/280#discussioncomment-592496.
